RadarSDK/RadarSdkConfiguration.h

```diff
@@ -0,0 +1,43 @@
+//
+//  RadarSdkConfiguration.h
+//  RadarSDK
+//
+//  Copyright © 2023 Radar Labs, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "Radar.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ Represents server-side sdk configuration.
+ 
+ @see https://radar.com/documentation/sdk/ios
+ */
+@interface RadarSdkConfiguration : NSObject
+
+@property (nonatomic, assign) RadarLogLevel logLevel;
+
+/**
+ Initializes a new RadarSdkConfiguration object with given value.
+ */
+- (instancetype)initWithLogLevel:(RadarLogLevel)logLevel;
```

Comment:
Do we want to call it this? It seems to me that the log level is unlikely to remain the most important field in this obj when we are done.